src/API.jl

```diff
@@ -709,6 +709,22 @@ instantiate(; kwargs...) = instantiate(Context(); kwargs...)
 function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing,
                      update_registry::Bool=true, verbose::Bool=false, kwargs...)
     Context!(ctx; kwargs...)
+    if !isfile(ctx.env.project_file) && isfile(ctx.env.manifest_file)
+        if manifest == false
+            pkgerror("cannot instantiate without a project file with `manifest=false` option")
+        end
+        _manifest = Pkg.Types.read_manifest(ctx.env.manifest_file)
+        deps = Dict()
+        for (uuid, pkg) in _manifest
+            if pkg.name in keys(deps)
+                # TODO, query what package to put in Project when in interactive mode?
+                pkgerror("cannot instantiate a manifest without project file when the manifest has multiple packages with the same name ($(pkg.name))")
+            end
+            deps[pkg.name] = string(uuid)
+        end
+        Types.write_project(Dict("deps" => deps), ctx.env.project_file)
+        return instantiate(Context(); manifest=manifest, update_registry=update_registry, verbose=verbose)
```

Comment:
```suggestion
        return instantiate(Context(); manifest=manifest, update_registry=update_registry, verbose=verbose, kwargs...)
```

Reply:
The kwargs are already applied to the context though?

Reply:
But this line creates a new context as well, it doesn't seem that e.g. `io=stdout` would go through

Reply:
Oh yeah...